### PR TITLE
Add wwwroot to .gitignore

### LINKED ISSUE #9184: Update .gitignore

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #9184

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# The 5 approved on boarding samples that need wwwroot can add a local .gitignore that allows wwwroot.
```
Comment (tdykstra) [inline .gitignore]: The change in https://github.com/aspnet/Docs/pull/4532 removed ignore lines for wwwroot/lib, not all of wwwroot.  Are you sure it won't create problems to ignore all of wwwroot?  Don't some projects put project-specific files in there on occasion?

**Comment by Rick-Anderson** [comment]
[This version of .gitignore](https://github.com/aspnet/Docs/commit/29d43601f87a883f0beade5b7231097ed9c4a15f) had wwwroot

**Comment by Rick-Anderson** [comment]
@danroth27 	can we merge this? We've had wwwroot listed in *.gitignore* in the past. If it's turns out to be a problem we'll update it.

**Comment by danroth27** [comment]
@Rick-Anderson No, I wouldn't. This *.gitignore* file change seems way too aggressive. I'm sure we will have some sample apps that need content *wwwroot* or will want to include JavaScript or CSS files. I would instead focus our efforts on simplifying the existing samples

**Comment by Rick-Anderson** [comment]
Last pitch. It seems aggressive but it's actually not. We've had *wwwroot* in for extended periods with no problems. We've almost always had `wwwroot/lib/` in *.gitignore*.

For the 95% of samples that don't need *wwwroot*, it's alot of work to remove  *wwwroot* and it's typically not done. It's impossible to police keeping out unneeded *wwwroot* assets without *.gitignore*.

For the 5% that need *wwwroot* , it's trivial to add.

Should we go back to what we had for most of our history? see #4532

```
**/sample/**/wwwroot/lib/
**/samples/**/wwwroot/lib/
```

I'm OK with `wwwroot/lib/` - that's 97% of `wwwroot`.

Keep in mind samples that currently contain *wwwroot* assets don't loose them when *wwwroot* is added back and the project is updated.

**Comment by danroth27** [comment]
> Should we go back to what we had for most of our history? see #4532

I think that's certainly better than excluding all static files support from every sample.

**Pull request merged**